Resources/Prototypes/_Moffstation/PersonalItems/Items/Compilatron/Tania Buttersworth/stationblueprints.yml

```diff
@@ -0,0 +1,35 @@
+- type: entity
+  parent: BaseItem
+  id: PersonalItemStationBlueprints
```

Comment:
Name should be something like `PersonalItemTaniaStationBlueprints` or `PersonalItemSharedStationBlueprints`